cumulusci/tasks/metadeploy.py

```diff
@@ -24,6 +24,7 @@ class BaseMetaDeployTask(BaseTask):
     """Base class for tasks that talk to MetaDeploy's API."""
 
     def _init_task(self):
+        assert self.project_config.keychain
```

Comment:
This line just proves that the next line won't crash, so it should be backwards compatible.